exporter/exporterparser/opencensus.go

```diff
@@ -72,28 +78,38 @@ func OpenCensusTraceExportersFromViper(v *viper.Viper) (tes []exporter.TraceExpo
 		opts = append(opts, ocagent.WithHeaders(ocac.Headers))
 	}
 
-	sde, serr := ocagent.NewExporter(opts...)
-	if serr != nil {
-		return nil, nil, nil, fmt.Errorf("Cannot configure OpenCensus Trace exporter: %v", serr)
+	numWorkers := defaultNumWorkers
+	if ocac.NumWorkers > 0 {
+		numWorkers = ocac.NumWorkers
+	}
+
+	exporters := make([]*ocagent.Exporter, 0, numWorkers)
+	for exporterIndex := 0; exporterIndex < numWorkers; exporterIndex++ {
+		exporter, serr := ocagent.NewExporter(opts...)
+		if serr != nil {
+			return nil, nil, nil, fmt.Errorf("Cannot configure OpenCensus Trace exporter: %v", serr)
+		}
+		exporters = append(exporters, exporter)
+		doneFns = append(doneFns, func() error {
+			exporter.Flush()
+			return nil
+		})
 	}
 
-	oexp := &ocagentExporter{exporter: sde}
+	oexp := &ocagentExporter{exporters: exporters}
 	tes = append(tes, oexp)
 
 	// TODO: (@odeke-em, @songya23) implement ExportMetrics for OpenCensus.
 	// mes = append(mes, oexp)
-	doneFns = append(doneFns, func() error {
-		sde.Flush()
-		return nil
-	})
 	return tes, mes, doneFns, nil
 }
 
-func (sde *ocagentExporter) ExportSpans(ctx context.Context, td data.TraceData) error {
-	err := sde.exporter.ExportTraceServiceRequest(
+func (oce *ocagentExporter) ExportSpans(ctx context.Context, td data.TraceData) error {
+	err := oce.exporters[rand.Intn(len(oce.exporters))].ExportTraceServiceRequest(
```

Comment:
Can you compare this versus a simple round-robin instead? Just thinking that under load you may get a sequence of requests going to a single exporter...